app/controllers/concerns/sufia/dashboard_controller_behavior.rb

```diff
@@ -0,0 +1,63 @@
+module Sufia
```

Comment:
Better to place the code into app/controllers/dashboard_controller.rb so to avoid many lines of duplicated code.